xla/backends/gpu/codegen/triton/fusion_emitter_legacy_matmul.cc

```diff
@@ -227,121 +227,6 @@ bool IsFp8Type(Type t) {
                    mlir::Float8E4M3B11FNUZType>(t);
 }
 
-Value Cast(EmitterLocOpBuilder b, Value value, Type dst_element_ty) {
```

Comment:
This function (along with other functions as well) was duplicated in `emitter_helpers.{h,cc}`.
I did not see a reason for this, so I refactored this function to always use the version from `emitter_helpers` as a drive-by.

Maybe a broader refactor is in place here, to eliminate duplicated code?